### PR TITLE
No longer use the deprecated forecast attribute, fixing the deprecation warning.

### LINKED ISSUE #26: Warning: The meteoswiss integration is using deprecated forecast. This stops working in version 2024.3.0
Just FYI...

### PR DESCRIPTION
**Pull request by mikispag**
Fixes #26.

### CODE REVIEW

**Review by vistalba — APPROVED** [review]
lgtm. Tested on my HA instance.

File: custom_components/meteoswiss/weather.py [hunk 1]
```diff
@@ -68,7 +64,7 @@ def __init__(
     def __set_data(self, data: MeteoSwissClientResult) -> None:
         self._displayName = data[CONF_FORECAST_NAME]
         self._forecastData = data["forecast"]
-        self._condition = data["condition"]
+        self._condition = data["condition"][0]
```
Comment (Rudd-O) [inline custom_components/meteoswiss/weather.py:67]: This sometimes fails.

I've implemented a fix for this in master.  Please rebase onto master and I will merge this.

**Review by Rudd-O — CHANGES_REQUESTED** [review]

File: custom_components/meteoswiss/weather.py (hunk 1 above)
Comment (mikispag) [reply custom_components/meteoswiss/weather.py:90]: Done.

Comment (mikispag) [reply custom_components/meteoswiss/weather.py:90]: Shall I call `condition_name_to_value` on `data["condition"]` or is it fine now?

**Comment by mikispag** [comment]
Thanks. PTAL.

**Review by Rudd-O — APPROVED** [review]
This is truly awesome.  Thank you @mikispag for solving this issue!

**Pull request merged**